keras_cv/layers/preprocessing/cut_mix.py

```diff
@@ -12,13 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import tensorflow as tf
-import tensorflow.keras.layers as layers
 
 from keras_cv.utils import fill_utils
 
 
 @tf.keras.utils.register_keras_serializable(package="keras_cv")
-class CutMix(layers.Layer):
+class CutMix(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
```

Comment:
Should we override _augment and tf.Assert(False)?  This layer should never be called with 1 image.

Reply:
This ensures we only use _batch_augment